dbms/src/Server/CMakeLists.txt

```diff
@@ -8,10 +8,10 @@ add_library(clickhouse-server
     StatusFile.cpp
     ReplicasStatusHandler.cpp
 )
-target_link_libraries(clickhouse-server daemon storages_system)
+target_link_libraries(clickhouse-server daemon storages_system clickhouse_functions)
```

Comment:
There is little discrepancy: now we have different styles:
- in `dbms` library, ClickHouse codebase is called `dbms`;
- in `storages_system` library, there is no any mention of ClickHouse;
- in clickhouse_functions library, ClickHouse codebase is called `clickhouse`.